NOTICE.md

```diff
@@ -181,7 +181,7 @@ SOFTWARE.
 
 ```
 
-## {fmt}
+## &#x7b;fmt&#x7d;

```

Comment:
lemme guess, this is because `ConvertFrom-Markdown` explodes on the `{}`?

Reply:
It actually deleted the `{fmt}` entry _entirely_ when i converted it o_O